src/main/handlers.js

```diff
@@ -326,24 +326,28 @@ function installLinux () {
   function writeIconFile (err, iconFile) {
     if (err) return log.error(err.message)
 
+    const mkdirp = require('mkdirp')
+
     const iconFilePath = path.join(
       os.homedir(),
       '.local',
       'share',
       'icons',
       'webtorrent-desktop.png'
     )
-    fs.mkdirp(path.dirname(iconFilePath))
-    fs.writeFile(iconFilePath, iconFile, function (err) {
```

Comment:
There was a race condition here where `iconFilePath` might not be created before `fs.writeFile` tries to write into it. Fixed.